recipe/meta.yaml

```diff
@@ -25,11 +25,13 @@ requirements:
     - setuptools
   run:
     - python >=3
+    - setuptools
     - pytz
     - tzlocal
     - requests
     - certifi
     - bagit
+    - bagit-profile
```

Comment:
The `bagit-profile` module is a new dependency with `bdbag-1.6.0` that is up on PyPi but doesn't have a corresponding Conda package. The condaforge build fails trying to find a Conda version of this dependency. I don't expect a Conda version of this to happen any time soon if at all. Not sure how to proceed.

Reply:
I opened a conda-forge recipe PR for `bagit-profile` [here](https://github.com/conda-forge/staged-recipes/pull/13936). We'll see what happens.